assets/pages/Operator.jsx

```diff
@@ -102,6 +104,22 @@ export default class Operator extends Component {
     this.forceUpdate()
   }
 
+  onProfileOpen = () => {
+    this.setState({
+      profileOpen: true,
+    })
+  }
+
+  onProfileClose = () => {
+    this.setState({
+      profileOpen: false,
+    })
+  }
+
+  onProfilePick = (data) => {
+    alert(JSON.stringify(data.rowData))
```

Comment:
No, that is wrong! We expect just a MODEL from our picker – not any other data. You should process this inside your picker component and return just a model. This is described in issue, and also we discussed this moment.